bigchaindb/web/views/validators.py

```diff
@@ -0,0 +1,24 @@
+from flask import current_app
+from flask_restful import Resource
+
+from bigchaindb.web.views.base import make_error
+
+
+class ValidatorsApi(Resource):
+    def get(self):
+        """API endpoint to get validators set.
+
+        Return:
+            A JSON string containing the validator set of the current node.
+        """
+
+        pool = current_app.config['bigchain_pool']
+
+        try:
+            with pool() as bigchain:
+                validators = bigchain.get_validators()
+
+            return validators
+
+        except Exception:
```

Comment:
:warning: catch the exceptions you expect may happen :warning: 

A generic `except Exception` will catch **everything**:
```
>>> issubclass(NameError, Exception)
True

>>> issubclass(SyntaxError, Exception)
True
```

A system that crashes because we didn't catch an exception is easier to debug than one that suppresses **any** exception.

Also, if the exception is not catch I think `flask` is returning a `500` anyway.

Reply:
Ideally the only exception I expect is a connection to the Tendermint's HTTP API. In case of such connection error the client should still see `500` which implies that we don't need to handle any exceptions here as `flask` by default is returning `500` anyway. The addendum I made now is to handle the connection exception and create a log message for easier debugging.